backend/src/labelit/models/document.py

```diff
@@ -10,6 +10,9 @@ class Document(models.Model):
         blank=True,
         null=True,
     )
+    video_filename = models.CharField(
```

Comment:
Another solution would be to add an enum field called `type` which reference the type of the document. Then rename `audio_filename` by `filename`. This allows to easily add new document type in the future. I don't know what do you think about this ?